README.md

```diff
@@ -1,10 +1,91 @@
+
+Happy Path is based on these principles:
+
+* Operational simplicity: Happy Path takes the bite out of a complex container orchestration operations and infrastructure management
+* Extensibility: Happy Path functionality can be extended through various Terraform hooks, custom Terraform modules and Helm Chart customization
+* Reliability: Happy Path is reliable and is production-ready, used by multiple engineering teams at CZI
 
 ### Security
 
 Please note: If you believe you have found a security issue, please responsibly disclose by contacting us at security@chanzuckerberg.com
 
+Visit the Happy Path documentation for more details: https://chanzuckerberg.github.io/happy/
+
+### Repository structure
+
+This project is a monorepo for the components of a Happy ecosystem:
+* `cli/` - the `happy` CLI tool
+* `api/` - the `happy` API server
+* `shared/` - components shared between `api` and `cli`
+* `hvm/` - the `happy` version manager
+* `terraform/` - a collection of TF modules we use to provision long-lived infrastructure and application stacks
+* `helm-charts/stack` - an experimental helm chart
+* `examples/` - sample applications that illustrate various concepts that `happy` supports, such as sidecars, tasks, multi-service deployments, GPU intense workloads, and so on
+
+
+### Features
+* Manages short-lives infrastructure (we deploy into your compute)
+* Groups services together (we call it a `stack` for co-deployment), each `stack` is isolated, and you can have multiple `stacks` created for the same application.
+* Easily promote changes from lower to higher environments
+* Supports automated deployments through github workflows
+* Has an extensive set of Github workflows
+* Supports both AWS ECS and EKS runtimes, and allows for an easy migration between the two
+* Abstracts out IaC code with the intent that developers should only focus on the application code
+* Supports Linkerd service mesh, mTLS and service-to-service authorization when deployed on EKS with Linkerd installed
+* Plays nicely with `external-dns`, `karpenter`, `cluster-autoscaler`
+* Integrates with Datadog for dashboarding (assuming you have a `datadog` agent deployed into your EKS)
+* Provides service discovery and autoscaling capabilities
+* Supports both amd64 and arm64 architectures
+* Supports metrics collection, health monitoring through healthchecks, and synthetics
+* Supports rolling updates to minimize downtime
+
+### Prerequisites
+
+You will need to have Docker desktop, AWS CLI, and `terraform` installed to use Happy.
+
+### Install
+
+Install `happy`:
+
+#### MacOS
+```sh
+brew tap chanzuckerberg/tap
+brew install happy
```

Comment:
Should we instruct people to use `hvm` instead?

Reply:
Please edit the details.

Reply:
~~~
hvm install chanzuckerberg happy 0.118.1
hvm set-default chanzuckerberg happy 0.118.1
~~~